File: CampingNeretva/CampingNeretva.Model/WorkerModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CampingNeretva.Model
{
    public class WorkerModel
    {
        public int WorkerId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public List<RoleModel> Roles { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
    }
}

File: CampingNeretva/UI/campingneretva_mobile/lib/models/worker_model.dart
class Worker {
  final int id;
  final String firstName;
  final String lastName;
  final String phoneNumber;
  final String email;

  Worker({
    required this.id,
    required this.firstName,
    required this.lastName,
    required this.phoneNumber,
    required this.email,
  });

  factory Worker.fromJson(Map<String, dynamic> json) {
    return Worker(
      id: json['workerId'],
      firstName: json['firstName'],
      lastName: json['lastName'],
      phoneNumber: json['phoneNumber'],
      email: json['email'],
    );
  }

  String get fullName => "$firstName $lastName";
}
